lib/controllers/Motor.h

```diff
@@ -44,9 +43,18 @@ class Motor {
    */
   float getPower();
 
+  // Servo type exclusive functions
+  bool servoSetRange(float range_);         // Returns FALSE if t_motorType != lim_servo
+  bool servoSetMaxSpeed(float max_speed_);  // Returns FALSE if t_motorType != cont_servo
```

Comment:
I'm still not 100% on board with the whole idea of having functions perform different operations depending on a type etc... where are these used? If you are going to include the servo within the motor class I would at least try to consolidate the functions with duplicate functionality. `setMaxSpeed`, `getMaxSpeed`, `setSpeed` are also applicable to a normal motor, but haven't you mapped this stuff already in the servomotor class?

Reply:
This also means that the actuator class will work without special modification for the servo

Reply:
I feel that normal motors, and continous servos should be in the same class / inherit (note similar functionality) though the poisitional servo really has no reason to be in the same class. What do you think?

Reply:
If what I commented doesn't make sense maybe we can have a short call, I may not have understood exactly what you did.

Reply:
Yeah I totally agree with this...my bad, forgot to remove these functions from the Motor class... they should only be present in the ServoMotor class